PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetExtractor.cxx

```diff
@@ -807,6 +810,12 @@ Bool_t AliAnalysisTaskJetExtractor::Run()
   if(fRandomGenerator->Rndm() >= fEventPercentage)
     return kFALSE;
 
+  if (fRejectTPCPileup)   {
+      Bool_t acceptEventCuts = fEventCuts.AcceptEvent(InputEvent());
+      if(!acceptEventCuts)     return kFALSE;
+    }
```

Comment:
You don't need this. `fEventCuts` evaluates with whatever settings you provided it in the base class